index.js

```diff
@@ -75,88 +95,126 @@
 //     name: "Liza", phones: { cell: "555-234-5678", office: "555-567-1234" }
 //   }
 // ]
+
+// function printName (name) {
+//   console.log( `${name}` )
+// }
+
+// users.forEach (printName,val)
```

Comment:
close! Here we can make a function which accepts an array of people as an argument. Then iterate over that array. For each userObject in our array, we will console log some information about that user.

```

function printNameAndPhones(usersArr) {
  usersArr.forEach(function(userObject) {
    console.log(userObject.name);
    console.log(`Cell: ${userObject.phones.cell}`)
    console.log(`Office: ${userObject.phones.office}`)
  })
}
```